tests/Components/Powertrain/test_SimpleMotor.py

```diff
@@ -0,0 +1,54 @@
+import attr
+from vec import Vector2
+from adr.World import Ambient
+from adr.Components import FreeBody
+from adr.Components.Powertrain import Motor, SimpleMotor
+from adr.Methods.Powertrain.thrust_equations import get_axial_thrust_from_linear_model
+import numpy.testing as npt
+import math
+import pytest
+
+@pytest.fixture
+def motor():
+    motor = SimpleMotor(
+       name='attached_component',
+       mass=1.4,
+       relative_position=Vector2(-0.4, 0.1),
+       relative_angle=math.radians(0),
+       static_thrust = 78,
+       linear_coefficient = 1,
```

Comment:
The linear coefficient is usually a negative value (thrust decays with an increase in velocity). There are some rare cases in which they increase, with special propellers, but that is not usually the case.